scss/_custom-styles.scss

```diff
@@ -109,6 +109,11 @@
   font-family: $font-family-sans-serif;
   font-weight: 500;
 }
+// >>> Material design icons.
+.material-icons {
+  font-size: inherit;
```

Comment:
It looks like this style is preventing utility classes (like `.display-3`) from working as expected; it's not allowing the class to change the font size.